feature_engine/feature_builder.py

```diff
@@ -27,7 +27,8 @@ def __init__(
             self, 
             input_file_path: str, 
             output_file_path_chat_level: str, 
-            output_file_path_conv_level: str
+            output_file_path_conv_level: str,
+            analyze_first_pct: float=1.0
```

Comment:
Note that, by default, `analyze_first_pct` is set to 1, so that we have backwards compatibility; running the previous models as-is leads to the same output.